### DISCUSSION TITLE
Ability to hotkey the "Move to next/previous section" buttons in Construction window

### DISCUSSION THREAD

**Discussion by RealTwentyEight** [OpenRCT2/OpenRCT2#23274] [Feature requests]
Would be awesome to hot key the "Move to next section" and "Move to previous section" buttons when building a ride.

![image](https://github.com/user-attachments/assets/b2afef46-77de-4984-8c9d-7390fa1d5662)

**Comment by telk5093** [comment]
![image](https://github.com/user-attachments/assets/8434c1f4-bf1f-414f-8176-f3eff49c30d6)

You may assign them in the config.

**Reply by RealTwentyEight** [reply]
I apparently didn't dig deep enough! Hell, googling for it returned nothing. Thanks!
